### ISSUE TITLE
AssignVariables - While conversion it is adding additional character ( ? )in the test due to which tests are failing.  - Bug

### ISSUE THREAD

**Issue by yajiv789** [apideck-libraries/portman#173]
Snippet of portman config - 
`"assignVariables": [
    {
      "openApiOperationId": "createOrg",
      "collectionVariables": [
        {
          "responseBodyProp": "id",
          "name": "createOrg.id"
        }
      ]
    }
  ],`

Postman collection test snippet:

// pm.collectionVariables - Set createOrg.id as variable for jsonData.id
```
if (jsonData?.id) {
   pm.collectionVariables.set("createOrg.id", jsonData.id);
   console.log("- use {{createOrg.id}} as collection variable for value",jsonData.id);
} else {
   console.log('INFO - Unable to assign variable {{createOrg.id}}, as jsonData.id is undefined.');
};
```

**Comment by yajiv789** [comment]
In the above postman test the if condition is incorrect ( addition special character [ ? ] is present )

**Comment by nicklloyd** [comment]
Hi @yajiv789 - that `?` is an optional chaining operator that ensures the jsonData is defined, and that it has an id.

If you run this request in Postman, can you verify the shape of the response, and check if there is output in console?

**Comment by yajiv789** [comment]
Here is the snapshot of postman test with special character:
![image](https://user-images.githubusercontent.com/18140429/143573575-6cfa3bf8-f14b-408d-90e1-51d697638d57.png)

Here is the snapshot of postman test without special character:
![image](https://user-images.githubusercontent.com/18140429/143573970-aca55f6f-5fab-4fdc-a000-f27795d8a62a.png)

**Comment by thim81** [comment]
@yajiv789 
Can you share on which OS you are working and which version of Postman?

**Comment by yajiv789** [comment]
![image](https://user-images.githubusercontent.com/18140429/143578492-65d92203-8e71-411c-83d3-0d3e9ea21309.png)

![image](https://user-images.githubusercontent.com/18140429/143578535-3ce1b05f-274f-44e8-be47-d161f839fe1b.png)

**Comment by yajiv789** [comment]
Postman Version - 7.36.5
Os - macOS Monterey - Version 12.0.1

**Comment by thim81** [comment]
We are using the latest postman 9.2.0, which might have a different NodeJS runtime.

If you run the tests with ?, in Newman do they fail as well?

**Comment by yajiv789** [comment]
Yes - In Newman ( Version - 5.3.0 ) also they fail 

![image](https://user-images.githubusercontent.com/18140429/143602801-e4d4795d-294c-4f33-8c33-2eda61923365.png)

**Comment by thim81** [comment]
Would you mind sharing the generated collection, so we can load in our Postman app?
It is a very special issue, which we want to better understand and try to reproduce.

Also, which version of NodeJS have you installed?

**Comment by thim81** [comment]
Investigation pointed out that "?" Optional chaining is only supported since NodeJS 14. 

We will look into how to solve this. If you want to continue without waiting, we would suggest updating to NodeJs 14.

**Comment by yajiv789** [comment]
I am using node version - v12.8.1
Sure, will share.

**Comment by thim81** [comment]
@yajiv789 We just released 1.10.2, which should fix your issue reported. 

If you are using a Node version, lower than Node 14, Portman will not use "optional chaining" to generated Postman collection.

**Comment by yajiv789** [comment]
@thim81 Thanks alot for your prompt responses and quick turn around with fix.
